cmd/node/node.go

```diff
@@ -554,11 +550,16 @@ func (app *SpacemeshApp) initServices(ctx context.Context,
 	atxdb := activation.NewDB(atxdbstore, idStore, mdb, layersPerEpoch, goldenATXID, validator, app.addLogger(AtxDbLogger, lg))
 	tBeaconDB := tortoisebeacon.NewDB(tBeaconDBStore, app.addLogger(TBeaconDbLogger, lg))
 
-	// TODO(nkryuchkov): Enable weak coin when finished.
-	// wc := weakcoin.NewWeakCoin(weakcoin.DefaultThreshold(), nodeID, swarm, signing.VRFVerify, vrfSigner, app.addLogger(WeakCoinLogger, lg))
-	wc := weakcoin.ValueMock{Value: false}
+	wcConfig := weakcoin.DefaultConfig()
```

Comment:
What do you think of something like this?
```suggestion
	wcConfig := weakcoin.DefaultConfig(weakcoin.WithMaxRound(app.Config.TortoiseBeacon.RoundsNumber))
```

Reply:
i don't like adding options to config. it would be a bit messy. i can get rid from config completely and just pass WithMaxRound to weakcoin.New